include/RAJA/internal/exec-openmp/scan_openmp.hxx

```diff
@@ -90,17 +90,17 @@ void inclusive_inplace(const ::RAJA::omp_parallel_for_exec&,
 {
   using Value = typename ::std::iterator_traits<Iter>::value_type;
   const int n = end - begin;
-  const int p = omp_get_max_threads();
+  const int p = std::min(n, omp_get_max_threads());
   ::std::vector<Value> sums(p, Value());
-#pragma omp parallel
+  #pragma omp parallel num_threads(p)
```

Comment:
Can you get rid of the indent on the pragmas, so that we are consistent with the rest of the code.